adapters/types.go

```diff
@@ -0,0 +1,36 @@
+package adapters
```

Comment:
It would be better to rename types.go -> delete_condition.go

Reply:
changed